### DISCUSSION TITLE
collections.all in (reverse) order, on Cloudflare Pages –– not working correctly

### DISCUSSION THREAD

**Discussion by michelbaas** [11ty/eleventy#2458] [Q&A] [answered]
So I'm using the following Nunjucks to generate a list of the last three recent publications/posts.
`{% set postslist = collections.all | head(-3) | reverse %}`

And it's working fine locally, both with the development as with the build version.
But as soon as it's pushed to Cloudflare, the order is almost like random. Definitely not in reverse, with the most recent post on the left or on top of the list.

What could be causing this?

**Comment by zachleat** [comment, marked as answer]
Not at the computer atm but check the content dates docs for CI server sort order

**Reply by michelbaas** [reply]
Thank you. I was relying on file creation dates. Adding the ISO 8601 format YYYY-MM-DD to the frontmatter resolved the issue.

**Reply by zachleat** [reply]
Great! For other folks the docs for this are here: https://www.11ty.dev/docs/dates/#collections-out-of-order-when-you-run-eleventy-on-your-server
